lib/inspec/rspec_json_formatter.rb

```diff
@@ -105,50 +106,36 @@ def initialize(*args)
     @backend = nil
   end
 
+  attr_reader :profiles
```

Comment:
Do we need to expose profiles? If not imho it's great to leave it private. What do you think?